src/backend/marsha/core/api/video.py

```diff
@@ -288,7 +288,8 @@ def perform_update(self, serializer):
             )
 
         channel_layers_utils.dispatch_video_to_groups(serializer.instance)
-        update_id3_tags(serializer.instance)
+        if (instance.is_live):
```

Comment:
this check should be added in the `update_id3_tags` as we are already testing video property in it ?